roles/identity-management/manage-rh-sso-identities/tasks/process_all_tasks.yml

```diff
@@ -0,0 +1,6 @@
+---
+
+- name: Process Red Hat SSO Identities
+  include_tasks: create_users.yml
```

Comment:
Can create/update logic be combined similar to how we want to do realms? Just add a user and make sure it looks like how we tell it to look, if they don't match -- then update?

Reply:
I have been thinking about it and my conclusion is that, because the user creation relies on `POST` HTTP method and the user update relies on `PUT`, I am assuming that they are meant to be executed as two distinct tasks.

A bit more on the topic: - Although we could implement a logic that updates the contents in case they don't match as we desire, this could potentially bring issues if we attempt to create but choose a username that is already in use. In this situation, the already existing user would get updated with the wrong values.

Based on that, we could definitely discuss it further and decide how we want to tackle this.

Reply:
Agreed with @tylerauerbeck on this ... would be good to make it a bit simpler by combining these two.